Masquerade4997CV/src/main/java/MasqCV/detectors/MasqCVDetector.java

```diff
@@ -111,4 +112,14 @@ public void onViewportTapped()
     public Size getSize() {
         return size;
     }
+
+    public Mat cropMat(Mat input, Point tl, Point br)  {
+        if (!(tl == null || br == null || tl.x >= input.width() || tl.y >= input.height() || tl.x < 0 || tl.y < 0 || br.x > input.width() || br.y > input.height() || br.x <= 0 || br.y <= 0)) {
+            Imgproc.rectangle(input,new Point(tl.x, tl.y),new Point(br.x, br.y),new Scalar(0), -1);
+        }
+        else {
+            DashBoard.getDash().create("Cropping failed due to invalid cropping margins");
+        }
+        return input;
```

Comment:
Where'd that come from?

Reply:
I edited one of the screenshots you sent me